cinema/tests/test_movie_api.py

```diff
@@ -120,7 +110,6 @@ def test_post_image_to_movie_list(self):
                 },
                 format="multipart",
             )
-
         self.assertEqual(res.status_code, status.HTTP_201_CREATED)
         movie = Movie.objects.get(title="Title")
         self.assertFalse(movie.image)
```

Comment:
In the `test_post_image_to_movie_list` method, the assertion `self.assertFalse(movie.image)` might be misleading if the intention is to check for the presence of an image. Consider revising this assertion to reflect the correct test intention.